recipes/gwcosmo/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gwcosmo" %}
+{% set version = "2.1.0a1" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Please see the instructions there on how to publish in a dev channel, or ask upstream for a stable release.